storage/innobase/log/log0write.cc

```diff
@@ -2713,7 +2715,16 @@ bool log_read_encryption() {
   redo_log_key *mkey = nullptr;
   Encryption::Type encryption_type = Encryption::NONE;
   uint version = 0;
-  if (memcmp(log_block_buf + LOG_HEADER_CREATOR_END, ENCRYPTION_KEY_MAGIC_RK,
+
+  if (memcmp(log_block_buf + LOG_HEADER_CREATOR_END, ENCRYPTION_KEY_MAGIC_RK_V1,
+               ENCRYPTION_MAGIC_SIZE) == 0) {
+    // can only happen during the upgrade
+    ib::error() << "Upgrade cannot continue as redo is encrypted with old, experimental version of KEYRING encryption. "
```

Comment:
this is old usage. ideal usage is ib::error(ER_BLAH);

Reply:
done